Skemc/Parking-Management-System
# #170749951 Integrate travis ci and test coverage in our project

#### What does this PR do?
Have CI and test coverage in our project 
#### Description of Task to be completed?
 - Integrate travis and coveralls with the project git repo
 - Add Travis CI and coveralls badges on the readme file
#### How should this be manually tested?
clone this repo and run the following commands
 - npm install
 - npm test


 >>Hereafter running the test you would be able to see the test coverage
Visit this repo(https://github.com/Skemc/Parking-Management-System) to see the badges
#### What are the relevant pivotal tracker stories?
#170749951
